index.php

```diff
@@ -8,17 +8,48 @@
 
 $plants = getData($db);
 
+$images = getImage($db);
+
+$plantImageArray = displayImage($images);
+
+$ids = getID($db);
+$id = displayID($ids);
+
 ?>
 
 <!DOCTYPE html>
 <html lang="eng">
 <head>
     <meta charset="utf-8">
     <title>Collectors App</title>
-    <link rel="stylesheet" type="text/css" href="index.css">
-    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="styles/index.css">
+    <link rel="stylesheet" type="text/css" href="styles/normalize.css">
 </head>
 <body>
+<div class="wrapper">
+    <section>
+        <div class="table">
+            <div>
+                <ul class="heading">
+                    <li>ID</li>
+                    <li>Common Name</li>
+                    <li>Latin Name</li>
+                    <li>Canopy Level</li>
+                    <li>Life Span</li>
+                </ul>
+            </div>
+                <?php echo processData($plants) ?>
```

Comment:
Charlie advised that the only functionality you should have in php tags within the html body is echoing string variables you already made in the php block. This method call should go up in the php block and load its message into a string, which you then echo here. This keeps your 'doing stuff' code all together.